CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to the VITAL Literature Review
+# Contributing to the VITALab website
 
```

Comment:
Maybe add a lower level header here? I understand that the file should remain general for the whole website, but right now we don't support contributions to anything but the literature reviews, and that's all the instructions linked to cover. So maybe something like:
```suggestion
# Contributing to the VITALab website

## Literature Reviews
```